metals/src/main/scala/scala/meta/internal/rename/RenameProvider.scala

```diff
@@ -86,7 +86,8 @@ final class RenameProvider(
         txtParams <- if (parentSymbols.isEmpty) List(textParams)
         else parentSymbols.map(toTextParams)
         currentReferences = referenceProvider.references(
-          toReferenceParams(txtParams)
+          toReferenceParams(txtParams),
+          includeSynthetics = false
```

Comment:
This will miss references to synthetic `.apply`, which we do want to rename

Reply:
right, I forgot to "de-focus" the tests locally. I'll think of a solution

Reply:
I've pushed a fix, which is the simplest thing I could think of: decide whether to include or not the synthetics based on the occurrence.

This way we can decide for which symbols we want to include the synthetics, e.g. we do want them for `apply`